desktop/api/src/main/java/org/datacleaner/panels/MutableInputColumnListPanel.java

```diff
@@ -51,7 +52,18 @@
     private final AnalysisJobBuilder _analysisJobBuilder;
     private final OutputColumnVisibilityButton _visibilityButton;
 
-    public MutableInputColumnListPanel(AnalysisJobBuilder analysisJobBuilder, MutableInputColumn<?> inputColumn) {
+    public JComponent panelOwner;
```

Comment:
OK that's fixed. But sorry it is still public and non-final. Any reason for that? Seems to me it should be private final.